cmd/nodecmd/destroy.go

```diff
@@ -93,12 +97,62 @@ func removeClustersConfigFiles(clusterName string) error {
 	return removeNodeFromClustersConfig(clusterName)
 }
 
-func CallDestroyNode(clusterName string) error {
+func CallDestroyNode(clusterName string, logs bool) error {
 	authorizeAll = true
+	enableLogs = logs
 	return destroyNodes(nil, []string{clusterName})
 }
 
+func getFirstAvailableNode(nodesToStop []string) (string, bool) {
+	firstAvailableNode := nodesToStop[0]
+	noAvailableNodesFound := false
+	for index, node := range nodesToStop {
+		nodeConfigPath := app.GetNodeConfigPath(node)
+		_, err := os.Stat(nodeConfigPath)
+		if err != nil {
+			if index == len(nodesToStop)-1 {
+				noAvailableNodesFound = true
+			}
+			if errors.Is(err, os.ErrNotExist) {
+				continue
+			}
+		}
+		firstAvailableNode = node
+	}
+	return firstAvailableNode, noAvailableNodesFound
+}
+
+func Cleanup() error {
```

Comment:
i think logic for destroy can be improved by loading all clusters and then filtering this list by cluster name provided by users or using this list for `--all` case

Reply:
to make one code to work in all cases